pkg/cmd/taskrun/cancel_test.go

```diff
@@ -242,7 +253,7 @@ func TestTaskRunCancel_v1beta1(t *testing.T) {
 				Status: duckv1beta1.Status{
 					Conditions: duckv1beta1.Conditions{
 						{
-							Status: corev1.ConditionTrue,
+							Status: corev1.ConditionUnknown,
```

Comment:
This should be changed since the TaskRun is in a running state, so its Condition status should be unknown.